pkg/graph/resolvers/plan_data_exchange_approach_helper.go

```diff
@@ -62,8 +61,7 @@ func SendDataExchangeApproachCompletedEmailNotification(
 	return nil
 }
 
-func SendDataExchangeApproachCompletedEmailNotifications(
-	ctx context.Context,
+func SendDataExchangeApproachMarkedCompleteEmailNotifications(
```

Comment:
Question, is the email in any way modified when we call this function? It not, I think it might make sense to use the BCC function to send one email to multiple users instead of build and send the email multiple times.

Reply:
Yeah, if it's the same email being sent to multiple users I then using `BCC` would be preferred to making many calls to the SMTP server